core/dronecore.h

```diff
@@ -48,26 +47,45 @@ class DroneCore
      * Supports connection: Serial, TCP or UDP.
      * Connection URL format should be:
      * - UDP - udp://[Bind_host][:Bind_port]
-     * - TCP - tcp://[Server_host][:Server_port]
-     * - Serial - serial://[Dev_Node][:Baudrate]
+     * - TCP - tcp://[Remote_host][:Remote_port]
+     * - Serial - serial://Dev_Node[:Baudrate]
      *
-     * Default URL : udp://:14540.
-     * - Default Bind host IP is localhost(127.0.0.1)
-     *
-     * @warning Serial connections are not supported on Windows (they are supported on Linux and macOS).
+     * Default URL : udp://0.0.0.0:14540.
+     * - Default Bind host IP is any local interface (0.0.0.0)
      *
      * @param connection_url connection URL string.
      * @return The result of adding the connection.
      */
-    ConnectionResult add_any_connection(const std::string &connection_url = DEFAULT_UDP_CONNECTION_URL);
+    ConnectionResult add_any_connection(const std::string &connection_url);
+
+    /**
+     * @brief Adds a UDP connection to the specified port number on 0.0.0.0 on any local interface.
+     *
+     * @param local_port The local UDP port to listen to (defaults to 14540, the same as mavros).
+     * @return The result of adding the connection.
+     */
+    ConnectionResult add_udp_connection(int local_port = DEFAULT_UDP_PORT);
 
     /**
-     * @brief Adds a UDP connection to the specified port number.
+     * @brief Adds a UDP connection to the specified port number and local interface.
      *
-     * @param local_port_number The local UDP port to listen to (defaults to 14540, the same as mavros).
+     * To accept only connections local connections of the machine, use 127.0.0.1, for any
+     * incoming connections, use 0.0.0.0.
+     *
+     * @param local_bind_ip The local UDP ip to listen to.
+     * @param local_port The local UDP port to listen to (defaults to 14540, the same as mavros).
```

Comment:
mavros is the package name, MAVROS is the product name. I would say this is an instance of MAVROS.